docs/source/api/deliveryservices_id_safe.rst

```diff
@@ -32,39 +32,33 @@ Request Structure
 .. table:: Request Path Parameters
 
 	+------+--------------------------------------------------------------------------------+
-	| Name |                      Description                                               |
+	| Name | Description                                                                    |
 	+======+================================================================================+
 	|  ID  | The integral, unique identifier of the :term:`Delivery Service` being modified |
 	+------+--------------------------------------------------------------------------------+
 
-:displayName: The :ref:`ds-display-name`
-:infoUrl:     An :ref:`ds-info-url`
-:longDesc:    The :ref:`ds-longdesc` of this :term:`Delivery Service`
-:longDesc1:   The :ref:`ds-longdesc2` of this :term:`Delivery Service`
-:longDesc2:   The :ref:`ds-longdesc3` of this :term:`Delivery Service`
-
-.. note:: All of these fields are optional; this ``PUT`` behaves more like a ``PATCH``
+:displayName: A string that is the :ref:`ds-display-name`
+:infoUrl:     An optional\ [#optional]_ string containing the :ref:`ds-info-url`
+:longDesc:    An optional\ [#optional]_ string containing the :ref:`ds-longdesc` of this :term:`Delivery Service`
+:longDesc1:   An optional\ [#optional]_ string containing the :ref:`ds-longdesc2` of this :term:`Delivery Service`
```

Comment:
why the removal of longDesc2? Should that be supported here?

Reply:
Perl didn't support it, it was incorrect documentation (if you look at #2455 it also didn't support it, but didn't change the documentation to reflect that).